trainingaddon/acceleratoraddon/web/src/de/hybris/training/constants/TrainingaddonWebConstants.java

```diff
@@ -21,4 +21,9 @@ private TrainingaddonWebConstants()
 	}
 
 	// implement here constants used by this extension
+	final static int TEN_PRODUCTS = 10;
+

```

Comment:
This is wrong, this is not modifiable via backoffice, add a new attribute to the component tho handle the number of products to display in storefront, in that ay this an be modified via backoffice